tests/test_views.py

```diff
@@ -4,34 +4,73 @@
 
 from flask import make_response
 from marshmallow import fields, Schema, post_load
+from marshmallow import EXCLUDE
 
 from flask_apispec.utils import Ref
 from flask_apispec.views import MethodResource
 from flask_apispec import doc, use_kwargs, marshal_with
 
+
```

Comment:
nitpick: Maybe add a comment here explaining why `unknown = EXCLUDE` is set for these.

Reply:
Ok, can be enough? 

```suggestion
# All the following schemas are set with unknown = EXCLUDE
# because part of a multiple schema input.
# This way none of them will raise errors for unknown fields handled by others
```